examples/mobile/UIComponents/components/interactive/coverflow/index.html

```diff
@@ -14,13 +14,13 @@ <h1>CoverFlow</h1>
 	<div class="ui-content">
 		<div class="ui-coverflow" data-effect="carousel">
 			<ul class="flip-items">
-				<li><img src="images/1.jpg" class="coverflow-image"></li>
-				<li><img src="images/2.jpg" class="coverflow-image"></li>
-				<li><img src="images/3.jpg" class="coverflow-image"></li>
-				<li><img src="images/4.jpg" class="coverflow-image"></li>
-				<li><img src="images/5.jpg" class="coverflow-image"></li>
-				<li><img src="images/6.jpg" class="coverflow-image"></li>
-				<li><img src="images/7.jpg" class="coverflow-image"></li>
+				<li><img src="images/1.jpg" width="150px" height="150px"></li>
```

Comment:
This is defined in style.css, right?
So how about referencing to it like other apps do
e.g.
https://github.com/Samsung/TAU/blob/tau_1.1/examples/wearable/UIComponents/contents/graphs/static_graph.html
`	<link href="../../css/style.css" rel="stylesheet" />`

Reply:
sure. no problem.